src/style.css

```
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: myFont1;
+  src: url(fonts/Avenir-Book.ttf);
+}
```

it's interesting, but better use descriptive values.
Plus, you need to specify weight and style of the fonts:
pay attention to `font-weight` of two fonts provided (300 and 600) and apply respectively
```suggestion
@font-face {
  font-family: "Avenir";
  font-weight: 300;
  font-style: normal;
  src: url("./fonts/Avenir-Book.ttf")
}
```